packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -573,6 +595,11 @@ def validate(data: Any) -> None:
                 "You may use `AbciAppDB.validate(your_data)` to validate your data for the `AbciAppDB`."
             )
 
+    @staticmethod
+    def deterministic(value: Iterable) -> Tuple:
+        """Converts an iterable to a sorted tuple."""
+        return tuple(sorted(value))
```

Comment:
I think this should not have a method. It's totally unrelated to the concerns of the object. If anything have a standalone method, but I think might be cleaner to do it in place.

Reply:
Yes, I agree. The only reason I added it here is that I thought it might become relevant in the future when we address [[Feature] Ensure order of elements before storing in db](https://trello.com/c/kyijw5h9). In the future, we could extend this method to support more types and also nested data. 

I'll remove it for now since it is irrelevant.

Reply:
Done on b813831dd314af80bcc981cc0996dcec4bfa7889.